cli/cmd/outputs.go

```diff
@@ -64,3 +66,46 @@ func (c *cliState) FormatJSONString(s string) (string, error) {
 	}
 	return string(pretty), nil
 }
+
+// Used to clean CSV inputs prior to rendering
+func csvCleanData(input []string) []string {
+	var data []string
+	for _, h := range input {
+		data = append(data, strings.Replace(h, "\n", "", -1))
+	}
+	return data
+}
+
+// Used to produce CSV output
+func renderAsCSV(headers []string, data [][]string) (error, string) {
+	csvOut := &strings.Builder{}
+	csv := csv.NewWriter(csvOut)
+
+	if len(headers) > 0 {
+		if err := csv.Write(csvCleanData(headers)); err != nil {
+			return errors.Wrap(err, "Failed to build csv output"), ""
+		}
+	}
+
+	for _, record := range data {
+		if err := csv.Write(csvCleanData(record)); err != nil {
+			return errors.Wrap(err, "Failed to build csv output"), ""
+		}
+	}
+
+	// Write any buffered data to the underlying writer (standard output).
+	csv.Flush()
+	return nil, csvOut.String()
```

Comment:
uhmm, shouldn't we check for errors after flushing the buffer? 
```suggestion
	csv.Flush()
	return csv.Error(), csvOut.String()
```

Reply:
Please, do change the order of the returned elements to be `return csvOut.String(), csv.Error()`